### ISSUE TITLE
Offense missed by Style/StringLiterals

### ISSUE THREAD

**Issue by jonas054** [rubocop/rubocop#1432]
Initially reported by @e2 in #1415 (which was a different problem and is now closed).

Running `rubocop -c example.yml example.rb` with the files

``` yaml
Style/StringLiterals:
  EnforcedStyle: double_quotes

Lint/LiteralInInterpolation:
  Enabled: false
```

``` ruby
foo "#{bar}#{'all'}"
foo "#{bar} #{'all'}"
```

yields the following output:

```
Inspecting 1 file
C

Offenses:

example.rb:2:15: C: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
foo "#{bar} #{'all'}"
              ^^^^^

1 file inspected, 1 offense detected
```

The same offense should have been found on both lines. Only one was found.
